backend/src/main/java/com/benine/backend/http/CameraIrisHandler.java

```diff
@@ -0,0 +1,56 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.camera.Camera;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.IrisCamera;
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created on 21-05-16.
+ */
+public class CameraIrisHandler extends CameraRequestHandler {
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
```

Comment:
Same here, it is pretty long. It doesn't really bother me but I want to have your opinion on why you didn't split this method.